src/buildSupport/GradleBuildSupport.ts

```diff
@@ -26,8 +26,8 @@ export class GradleBuildSupport extends BuildSupport {
       quarkusBinary: 'buildNative',
       wrapper: 'gradlew',
       wrapperWindows: 'gradlew.bat',
-      taskBeginsPattern: '^.*Starting a Gradle Daemon*',
-      taskEndsPattern: '^.*Quarkus .* started in .*\\. Listening on:*'
+      taskBeginsPattern: '^.*Task: ',
+      taskEndsPattern: '^.*(Quarkus .* started in .*\\.| ERROR .* Failed to start)'
```

Comment:
don't you need a dollar sign to capture the end of the group? It's unclear this matches (success)|(failure) patterns

Reply:
It matches the existence of either pattern and doesn't have to match the full line (by including a dollar sign). I just figured there's no point in saying ".*$" when simply omitting it implies the same thing.

Reply:
Done. Round brackets are now of form (success)|(failure) and that seems to work just fine.